### PR TITLE
Bump version to 8.1

### PR DESCRIPTION
**Pull request by KOTungseth**
Bumps the docs to v8.1.

### CODE REVIEW

**Review by gtback — APPROVED** [review]
LGTM!

File: conf.yaml [hunk 1]
```diff
@@ -71,7 +71,7 @@ contents_title:     Elastic Stack and Product Documentation
 #   <key>: &<variable> <value>
 # The keys don't really matter, but by convention the are the same as the variable.
 variables:
-  stackcurrent: &stackcurrent 8.0
+  stackcurrent: &stackcurrent 8.1
   stacklive: &stacklive [ master, 8.1, 8.0, 7.17 ]
 
   stacklivemain: &stacklivemain [ main, 8.1, 8.0, 7.17 ]
```
Comment (bmorelli25) [inline conf.yaml:77]: Should 8.1 be removed from `stacklive` and `stacklivemain`?

Comment (KOTungseth) [reply conf.yaml]: Oh probably! Good catch :)

**Comment by bmorelli25** [comment]
```
19:19:14 INFO:build_docs:Bad cross-document links:
19:19:14 INFO:build_docs:  /tmp/docsbuild/target_repo/html/en/apm/agent/java/1.x/aws-lambda.html contains broken links to:
19:19:14 INFO:build_docs:   - en/apm/guide/current/aws-lambda-extension.html#aws-lambda-variables
19:19:14 INFO:build_docs:  /tmp/docsbuild/target_repo/html/en/apm/agent/java/current/aws-lambda.html contains broken links to:
19:19:14 INFO:build_docs:   - en/apm/guide/current/aws-lambda-extension.html#aws-lambda-variables
```

Looking into it...

**Comment by bmorelli25** [comment]
@elasticmachine, run elasticsearch-ci/docs rebuild

**Pull request merged**